content/patterns/button/button-pattern.html

```diff
@@ -14,92 +14,93 @@
   </head>
   <body>
     <main>
-      <h1>Button Pattern</h1>
+      <h1>ボタン パターン</h1>
+      <!-- 翻訳元リビジョン: https://github.com/w3c/aria-practices/tree/d6cf9d8db772603a456e7d7ef52e409d39124301 -->
 
       <section id="about">
-        <h2>About This Pattern</h2>
+        <h2>このパターンについて</h2>
         <p>
-          A <a class="role-reference" href="#button">button</a> is a widget that enables users to trigger an action or event, such as submitting a form, opening a dialog, canceling an action, or performing a delete operation.
-          A common convention for informing users that a button launches a dialog is to append &quot;&#8230;&quot; (ellipsis) to the button label, e.g., &quot;Save as&#8230;&quot;.
+          <a class="role-reference" href="#button">ボタン</a> は、フォームの送信、ダイアログのオープン、アクションのキャンセル、削除操作などの利用者のアクションやイベントのトリガを可能にするウィジェットです。
```

Comment:
「performing a delete operation」の「performing」が抜けてるかもしれないです。
また「利用者のアクション」とするよりは、「アクションやイベントを利用者がトリガーできるようにする」といったニュアンスが近いのかなと。

```suggestion
          <a class="role-reference" href="#button">ボタン</a> は、フォームの送信、ダイアログのオープン、アクションのキャンセル、削除操作の実行などのアクションやイベントを利用者がトリガーできるようにするウィジェットです。
```